src/__tests__/utility-types.spec.ts

```diff
@@ -26,8 +26,82 @@ type D2<T> = ReadonlyArray<T>
 type E2<T> = ReturnType<() => T>
 type F2<T, U> = Record<T, U>
 `;
-  const result = compiler.compileDefinitionString(ts, { quiet: true });
-  expect(beautify(result)).toMatchSnapshot();
+it("should handle utility types", () => {
+  const result = compiler.compileDefinitionString(utilityTypes, {
+    quiet: true,
+  });
+  expect(beautify(result)).toMatchInlineSnapshot(`
```

Comment:
I didn't know about `.toMatchInlineSnapshot`

Reply:
Oh! It's a meaningful improvement to snapshot testing, at least to my mind. It doesn't hide all the things you're making assertions about away in a file you never read!